utils/cryptography/encryption.go

```diff
@@ -0,0 +1,54 @@
+package crytography
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"errors"
+	"io"
+	"os"
+
+	"golang.org/x/crypto/nacl/secretbox"
+)
+
+// EncryptionKey for encrypting secrets
+var EncryptionKey, _ = base64.StdEncoding.DecodeString(os.Getenv("ENCRYPTION_KEY"))
+
+// Both what the encryption key length should be
+var keyLength = 32
+
+// EncryptMessage uses SecretBox to encrypt the message
+func EncryptMessage(field []byte) (encrypted []byte, nonceString [24]byte, err error) {
+	var nonce [24]byte
+
+	// The key argument should be 32 bytes long
+	if len(EncryptionKey) != keyLength {
+		return nil, nonce, errors.New("Encryption Key is not the correct key length")
+	}
+
+	var encryptionKey [32]byte
+	copy(encryptionKey[:], []byte(EncryptionKey))
+
+	if _, err := io.ReadFull(rand.Reader, nonce[:]); err != nil {
```

Comment:
any reason not to use use rand.Read here?

Reply:
this was directly copied from the example documentation here, prefer if we keep it as is 😄 https://godoc.org/golang.org/x/crypto/nacl/secretbox

Reply:
sounds like `ReadFull` is slightly preferable, https://groups.google.com/forum/#!topic/golang-nuts/eFS5GN-en3w
> Code should probably call io.ReadFull.
> There's no security problem with the current code's behavior,
because the rand.Reader is a bufio.Reader, and it
behaves like io.ReadFull on ordinary reads.
But that is a detail of one implementation,
not something to rely on.